src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,71 @@
-import React from 'react';
+import React, { FormEvent, useState, useCallback } from 'react';
+import classNames from 'classnames';
+import { CommentData } from '../types/Comment';
+
+type PropsTypes = {
+  addComment: (comment: CommentData) => Promise<void>,
+  isInputUpdating: boolean
+};
+
+export const NewCommentForm: React.FC<PropsTypes> = ({
+  addComment,
+  isInputUpdating,
+}) => {
+  const [name, setNameValue] = useState('');
+  const [email, setEmailValue] = useState('');
+  const [body, setBodyValue] = useState('');
+  const [nameError, setNameError] = useState<boolean>(false);
+  const [emailError, setEmailError] = useState<boolean>(false);
+  const [bodyError, setBodyError] = useState<boolean>(false);
+
+  const setErrorsOnIputs = useCallback(() => {
+    setBodyError(true);
+    setNameError(true);
+    setEmailError(true);
+  }, []);
+
+  const clearErrors = () => {
+    setBodyError(false);
+    setNameError(false);
+    setEmailError(false);
+  };
```

Comment:
Duplicate code.
Siggesting to make just one fuction updateErrorsOnInputs
it can be something like that
```
const updateErrorsOnIputs = useCallback((value: boolean) => {
    setBodyError(value);
    setNameError(value);
    setEmailError(value);
  }, []);